### ISSUE TITLE
Portal List plugin seems broken

### ISSUE THREAD

**Issue by gaotongsh** [IITC-CE/ingress-intel-total-conversion#584]
Today I tried to install Portal List plugin but "Portal List" button did not appear. I found the following error on "About IITC" page:
![image](https://user-images.githubusercontent.com/12027397/182539618-ca731849-d331-4fb0-99b0-b7edd1f342fc.png)
Can anyone reproduce this bug or my plugin is broken locally?

**Comment by le-jeu** [comment]
apparently, portal list plugin got the last release update while your other plugins and the main script IITC didn't yet
did you upgrade portal list manually?

Update all official plugins and main IITC to fix this

**Comment by gaotongsh** [comment]
Thanks, fixed after updated main IITC.

>  did you upgrade portal list manually?

I enabled Portal List for the first time today, so I assume it got the latest version.
There's no clear error message about what's going on and how to fix this. Perhaps we could prompt a message to users to update IITC when an unknown error occurs.

**Comment by le-jeu** [comment]
for information, what do you use for IITC ? Button/TamperMonkey/... ?

**Comment by gaotongsh** [comment]
> for information, what do you use for IITC ? Button/TamperMonkey/... ?

Button, not TamperMonkey

**Comment by modos189** [comment]
> Button, not TamperMonkey

Did you use the supplied IITC main script and plugin rather than install it manually?

**Comment by gaotongsh** [comment]
> Did you use the supplied IITC main script and plugin rather than install it manually?

Yes, I got it from the Chrome plugin store. I'm using Edge by the way.

**Comment by modos189** [comment]
@gaotongsh No, that's not what I mean. Did you install these plugins or did you just click the "enable" checkbox in the IITC Button plugin list?

**Comment by gaotongsh** [comment]
> Did you install these plugins or did you just click the "enable" checkbox in the IITC Button plugin list?

I always used checkboxes in the Button plugin list. Did not manually install Portal List plugin. I did manually installed the Bannergress plugin because it's not in the Button list but that's the only one.
